.github/workflows/e2e-test-ci.yml

```diff
@@ -24,6 +24,7 @@ jobs:
       run: |
         docker-compose -f ci/docker-compose.yml up -d
         sleep 5
+        docker logs apisix-seed
```

Comment:
Do we need to test the log options via unit tests?

Reply:
I thought about it, but currently the log is initialized in the main function, there is no independent module, and there is no good way to write unit tests

Reply:
It's fine